### PR TITLE
[4.x] Add Contributing Documentation

### PR DESCRIPTION
**Pull request by tjrgg**
### Items Changed

- [ ] Commands
- [ ] Events
- [ ] Handlers
- [ ] Structures
- [ ] Tasks
- [x] Documentation
- [ ] Other: ______

### Description

- Add CODE_OF_CONDUCT
- Add CONTRIBUTING
- Add issue templates
- Add pull request template

### Issues

N/A

### Have You...?

- [x] I have read the [Contributing Guidelines](https://github.com/typicalbot/typicalbot/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [pull request template](https://github.com/typicalbot/typicalbot/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
- [x] I have checked open [pull requests](https://github.com/typicalbot/typicalbot/pulls) for upcoming features and fixes.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [support@typicalbot.com](mailto:support@typicalbot.com). All
```
Comment (nsylke) [inline .github/CODE_OF_CONDUCT.md]: Update this email to opensource@sylke.co

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- [Style Guides](#style-guides)
+  - [CHANGELOG](#changelog)
+  - [Git Commit Messages](#git-commit-messages)
+  - [JavaScript Style Guide](#javascript-style-guide)
+  - [Documentation](#documentation)
+- [Getting Help](#getting-help)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [support@typicalbot.com](mailto:support@typicalbot.com).
```
Comment (nsylke) [inline .github/CONTRIBUTING.md]: Update this email to opensource@sylke.co

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Keep commits self-contained. Avoid spreading a single change across multiple commits.
+- Reference issues and pull requests liberally after the first line.
+
+### JavaScript Style Guide
+
+All JavaScript code must adhere to the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript), with the exception of [indents (19.1)](https://github.com/airbnb/javascript#whitespace--spaces). Use soft tabs (space character) set to 4 spaces.
+
+### Documentation
+
+All code must be documented using [ESDoc tags](https://esdoc.org/manual/tags.html).
```
Comment (nsylke) [inline .github/CONTRIBUTING.md:103]: Remove this section - I removed esdoc from the current version and I don’t know if I want to use it for the new version.

Comment (tjrgg) [reply .github/CONTRIBUTING.md:103]: Are we not going to replace it with something?

**Pull request closed without merging**